### ISSUE TITLE
`ThisWindow` thinks FvwmIconMan is always in a raised state

### ISSUE THREAD

**Issue by ThomasAdam** [fvwmorg/fvwm3#467]
Original post from:  https://fvwmforums.org/t/thiswindow-thinks-fvwmiconman-is-always-in-a-raised-state/3093

With this tiny `~/.fvwm/config` (fvwm3, latest master, f1f7c4e):

~~~
Module FvwmIconMan

Key F1 A A Next ("FvwmIconMan") my_test

DestroyFunc my_test
AddToFunc my_test
+ I ThisWindow (Raised) echo my_test: $[w.name] is raised
~~~

when I start fvwm3 (with -v option), open xterm, place it over FvwmIconMan window & press F1, I get this message in the log file:

    my_test: 'FvwmIconMan' is raised

`FvwmIconMan` should be be reporting itself as being raised.

![fvwm3-iconman](https://user-images.githubusercontent.com/101281/111867997-6a233700-896f-11eb-9f53-9d3b3b480149.png)

**Comment by somiaj** [comment]
I tried to play around with this a bit, and what I found out is FvwmIconMan is not triggering `VisibilityNotify` events except when it is created. So due to this the visibility state is determined by the visibility at the time the window is mapped and never changes. I was not able to figure out why FvwmIconMan is not triggering `VisiblityNotify` events after its creation.

https://tronche.com/gui/x/xlib/events/window-state-change/visibility.html
